code/src/app/shared/services/object-recognition.service.ts

```
@@ -0,0 +1,62 @@
+import { Injectable } from '@angular/core';
+
+import * as cocoSSD from '@tensorflow-models/coco-ssd';
+import * as mobilenet from '@tensorflow-models/mobilenet';
+import '@tensorflow/tfjs-backend-cpu';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ObjectRecognitionService {
+  constructor() {}
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)